doc/m3/FS22-Group-36-M3-Report.md

```diff
@@ -0,0 +1,46 @@
+\centering
+
+# SCREW YOUR NEIGHBOR
+
+Report for Milestone 3 of Sopra FS22, Group 36
+
+![cover (source B. Furrer)](../header/jass-image.png)
+
+**Members**
+
+Carmen Kirchdorfer (20-720-132)  
+Salome Wildermuth (10-289-544)  
+Beat Furrer, group leader (07-542-392)
+Lucius Bachmann (11-060-274)  
+Moris Camporesi (19-764-349)
+
+\newpage
+\raggedright
+## Diagrams
+
+### Database Schema
+![db_schema](uml/db_schema.svg){height=60%}
+
+
+### Class Diagram
+The class diagram that we handed in for M2 was quite sparse what was pointed out in the feedback. It has grown
+significantly (and also changed a bit) meanwhile the development process and because we use it permanently as a basis
+for our common understanding, we decided to hand in the extended version again with the report for M3.
+![class_diagram](uml/class_diagram.svg){height=60%}
+
+
+## UI Screenshots
+
+
+## Tests
+### Complex unittest
+Vorschlag 1: the_first_player_must_play_a_card_when_round_starts in HandTurnActiveTest
+Vorschlag 2: get_multiple_trick_winner_test in RoundTest
+
+### Integrationtest
+Vorschlag: play_last_card_new_round_new_match in CardEventHandlerTest
```

Comment:
CardEventHandlerTest kann man nehmen. 
Da müsste man wahrscheinlich für does_not_create_new_match_when_last_card_played_for_last_match noch das erstellen der vielen matches in einen for loop packen